package.json

```diff
@@ -18,7 +18,7 @@
     "publish": "pika publish",
     "version": "npm run build",
     "test": "npm run test:integration && npm run test:babel-plugin",
-    "test:integration": "jest test/integration/runner.js  --test-timeout=15000 --testMatch \"**/test/**/*.[jt]s?(x)\"",
+    "test:integration": "jest test/integration/runner.js --test-timeout=15000 --testMatch \"**/test/**/*.[jt]s?(x)\" -i",
```

Comment:
`-i` sped up our tests a bit, and will guarantee that tests run serially vs. in parallel (this will become more important now that we have a local cache to worry about)